src/stdx.zig

```diff
@@ -141,7 +141,6 @@ test "disjoint_slices" {
     try std.testing.expectEqual(false, disjoint_slices(u32, u8, b, std.mem.sliceAsBytes(b)));
 }
 
-// TODO(Performance): Iterate over words.
```

Comment:
Let's add a godbolt link (I tihnk one can get a shortened one?) as a comment here, as every person will be wondering how well this vectorizes. For godbolt, 

```

pub export fn f(bytes_ptr: [*]const u8, bytes_len: usize) bool {
    return zeroed(bytes_ptr[0..bytes_len]);
}
```

might be a simpler trick to cause the compiler to generate code, without including the whole main with the allocator and such. 

What we could perhaps do here as well is to take `[]const T`, and then take advantage of `T`'s size/alignment to generate tighter code wioutht `_split`, but yeah, this stone probably doesn't warrant more squeezing.

Reply:
I added a godbolt link. Good tip on `export fn` it's a lot cleaner.